R/epi_df.R

```diff
@@ -159,7 +161,8 @@ new_epi_df = function(x = tibble::tibble(), geo_type, time_type, as_of,
   
   # Reorder columns (geo_value, time_value, ...)
   if(sum(dim(x)) != 0){
-    x = dplyr::relocate(x, "geo_value", "time_value")
+    # Moves columns in place
+    setcolorder(x, c("geo_value", "time_value"))
```

Comment:
`new_epi_df` is part of our copy-on-write-compatible interface, so I don't think we can use `setcolorder` here without potentially introducing surprises in user code.

Reply:
(All `epi_df` stuff is expected to have this interface.  We only have the dual interface for `epi_archive`.  But we want to get rid of it as well and have only a COW-compatible one.)

Reply:
What do you think about [this approach](https://github.com/cmu-delphi/epiprocess/pull/386/commits/e00dc125415b6cdc30a4e076a3786839d1bc820b)? It's about the same speed as `data.table::setcolorder`; I changed to `setcolorder` for brevity.